Satzung.md

```diff
@@ -0,0 +1,186 @@
+
+(3) Die Sitzungen sind zu protokollieren. Die Protokolle sollen Ort und Zeit der Sitzung, die Namen der teilnehmenden Personen, die gefassten Beschlüsse und deren Abstimmungsergebnisse enthalten. Die Protokolle sind von der Sitzungsleitung zu unterschreiben. Jedes Vereinsmitglied ist berechtigt, die Protokolle einzusehen.
+
+
+## § 13 Aufgaben und Einberufung der Mitgliederversammlung
+
+(1) Die Mitgliederversammlung hat folgende Aufgaben:
+
+1. Genehmigung des vom Vorstand aufgestellten Haushaltsplans für das folgende Geschäftsjahr
+2. Entgegennahme des Jahresberichts des Vorstands
+3. Entlastung des Vorstands
+4. Wahl und Abwahl des Vorstands
+5. Änderungen der Satzung und Festlegung der Mitgliedsbeiträge
+6. Anträge des Vorstands und der Mitglieder
+7. Berufung/Beschwerde gegen die Ablehnung von Aufnahmeanträgen
+8. Ausschluss von Mitgliedern
+9. Vergabe und Aberkennung von Ehrenmitgliedschaften
+10. Auflösung des Vereins
+
+(2) Mitgliederversammlungen werden vom Vorstand in Textform, z.B. per E-Mail, unter Einhaltung einer Frist von einer Woche und Angabe der vorläufigen Tagesordnung einberufen. Einladungen gelten Mitgliedern als zugegangen, wenn sie an die letzte dem Verein in Textform bekanntgegebene E-Mail-Adresse gerichtet sind.
+
+(3) Der Vorstand muss jedes Jahr fristgemäß zu einer ordentlichen Mitgliederversammlung einladen, die an einem Tag im Zeitraum vom 25. bis zum 31. Januar stattfindet.
+
+(4) Der Vorstand kann jederzeit eine außerordentliche Mitgliederversammlung einberufen. Sie muss einberufen werden, wenn das Interesse des Vereins es erfordert oder wenn die Einberufung von einem Viertel aller Mitglieder in Textform, z.B. per E-Mail, unter Angabe der Gründe vom Vorstand verlangt wird. In letzterem Fall muss die außerordentliche Mitgliederversammlung spätestens vier Wochen nach Eingang des Antrags tagen.
+
+(5) Die Tagesordnung wird vom Vorstand festgesetzt. Jedes Mitglied des Vereins kann bis zum Beginn der Mitgliederversammlung beantragen, dass weitere Angelegenheiten, nicht jedoch der Ausschluss von Mitgliedern, Satzungsänderungen oder die Auflösung des Vereins, nachträglich auf die Tagesordnung gesetzt werden. Über die Annahme der Anträge beschließt die Mitgliederversammlung.
+
+
+## § 14 Beschlussfassung der Mitgliederversammlung
+
+(1) Die Mitgliederversammlung ist beschlussfähig, wenn mehr als die Hälfte der Mitglieder anwesend ist. Bei Beschlussunfähigkeit lädt der Vorstand umgehend zu einer zweiten Mitgliederversammlung mit gleicher Tagesordnung ein, die nach spätestens einem Monat tagt. Diese ist zu den in der Tagesordnung aufgeführten Punkten unabhängig von der Zahl der anwesenden Mitglieder beschlussfähig, wenn darauf mit der Einladung hingewiesen wird.
```

Comment:
```suggestion
(1) Die Mitgliederversammlung ist beschlussfähig, wenn mehr als die Hälfte der Mitglieder anwesend ist. Die Beschlussfähigkeit gilt ausschließlich für Tagesordnungspunkte, die zu Beginn der Mitgliederversammlung feststehen oder, im Falle von durch Mitgliedern beantragte Tagesordnungspunkten, zu Beginn der Versammlung akzeptiert werden. Bei Beschlussunfähigkeit lädt der Vorstand umgehend zu einer zweiten Mitgliederversammlung mit gleicher Tagesordnung ein, die nach spätestens einem Monat tagt. Diese ist zu den in der Tagesordnung aufgeführten Punkten unabhängig von der Zahl der anwesenden Mitglieder beschlussfähig, wenn darauf mit der Einladung hingewiesen wird.
```

Durch die Anmerkung soll sichergestellt werden, dass im Voraus Klarheit über die möglichen Beschlüsse, insbesondere Satzungsänderungen, herrscht. Da durch Mitglieder keine Satzungsänderungen spontan vorgeschlagen werden können, sollte es hier nicht zu Probleme kommen, allerdings kann es sinnvoll sein, diese Einschränkung explizit zu erwähnen.

Reply:
Ich bin mir nicht sicher, ob das implizit nicht sowieso klar ist. Im Endeffekt sagt dass doch nur, dass man für alle TO-Punkte beschlussfähig ist oder gibt es deiner Meinung nach noch TO-Punkte, die von deiner Ergänzung nicht erfasst werden?

Reply:
Stimmt, ist implizit schon eindeutig. Ich wollte es bloß mal in den Raum stellen, weil eine eindeutige Formulierung da später Diskussionen vermeiden könnte.

Reply:
Meiner Erfahrung nach, reicht es bei solchen Ordnungen, wenn es implizit klar ist, da es ja nichts an der Gültigkeit ändert. Dementsprechend würde ich den Satz rauslassen um die Ordnung etwas kompakter zu halten.